blog/saiit-wywiad/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Stowarzyszenie Autorów i Ilustratorów Technicznych (SAIiT) - Wywiad z Łukaszem Rybką'
+authors: mrozek
+date: '2024-06-30'
+tags:
+  - 'stowarzyszenie'
+  - 'zrzeszenia branżowe'
+  - 'promocja branży'
+coverImage: 'saiit.png'
+---
+
+Czy autorzy techniczni stanowią osobną grupę zawodową? 🤔 Dla wszystkich pracujących w tej branży z pewnością odpowiedź jest oczywista 💡 
+
+Okazuje się jednak, że poza branżą nie jest to już takie pewne. W Polsce Technical Writerzy nie są postrzegani w kategoriach samodzielnej profesji. Nasza praca często zaliczana jest do bardziej ogólnej kategorii: "usługi w sektorze IT".
+
+<!--truncate-->
+
+A to niesie za sobą pewne konsekwencje. 
+
+I o tym dziś będziemy rozmawiać z [Łukaszem Rybką](https://www.linkedin.com/in/%C5%82ukasz-rybka-364a81190/), pomysłodawcą i założycielem Stowarzyszenia Autorów i Ilustratorów Technicznych (SAIiT), który działa bardzo aktywnie, promując ideę stowarzyszenia wśród społeczności dokumentalistów. 
+
+## O inicjatorze
+
+#### Opowiedz nam o sobie w kilku zdaniach, jak długo zajmujesz się komunikacją techniczną, jaka była Twoja droga do tego zawodu i co teraz robisz?
```

Comment:
Teoretycznie "skipping heading levels" może być mylące dla użytkowników screen readerów. Dlatego specjaliści od accessibility zaleca nie pomijać poziomów. Po h2 powinien być h3.

W3 piszą na przykład:

> Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a `<h2>` is not followed directly by an `<h4>`, for example. It is ok to skip ranks when closing subsections, for instance, a `<h2>` beginning a new section, can follow an `<h4>` as it closes the previous section.

https://www.w3.org/WAI/tutorials/page-structure/headings/

Jeżeli chodzi o wizualny aspekt, można to rozwiązać na dwa sposoby:

1. Użyć customowego komponentu `<h3 style={{ fontSize: taki jak chcesz }}>`
2. Dodać h2, np. `## Treść wywiadu`, potem sekcje są h3, a pytania h4

Tak czy inaczej, proponuję się trzymać tej zasady nie pomijania poziomów.

Reply:
No właśnie... Celowo zrobiłam H4 żeby nie wyciągało mi H3 do side toc. Bo wygląda to tak, jeśli nie skipuję heading levels:

![image](https://github.com/techwriter-pl/techwriter-pl.github.io/assets/119131560/5fb55e0d-d90f-4e73-909c-3eda9da1b4d6)
 Co sądzisz? 

Spojrzałam też jak zrobił to @andrzejniewiaro w swoim wywiadzie z Oksaną, ale faktycznie chyba po prostu dał H2 - H3 i tyle.. 
![image](https://github.com/techwriter-pl/techwriter-pl.github.io/assets/119131560/c071dfb7-9014-4769-89ae-1d72bcac6dc6)

Reply:
Nie za dobrze to wygląda w right navie.

Można zmienić `toc_max_heading_level`< żeby się nie pokazywały H2.

https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#toc_max_heading_level

Reply:
Done! Dzięki! :)